```
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.022 s <<< FAILURE! -- in org.kie.yard.core.MVELJSONTest
[ERROR] org.kie.yard.core.MVELJSONTest.testMVELManagesJSONMaps -- Time elapsed: 0.019 s <<< ERROR!
java.lang.RuntimeException: interpretation failed at runtime
	at org.kie.yard.core.MVELLiteralExpressionInterpreter.fire(MVELLiteralExpressionInterpreter.java:59)
	at org.kie.yard.core.YaRDDefinitions.evaluate(YaRDDefinitions.java:43)
	at org.kie.yard.core.YaRDRunner.evaluate(YaRDRunner.java:58)
	at org.kie.yard.core.YaRDRunner.evaluate(YaRDRunner.java:63)
	at org.kie.yard.core.TestBase.evaluate(TestBase.java:34)
	at org.kie.yard.core.MVELJSONTest.testMVELManagesJSONMaps(MVELJSONTest.java:41)
Caused by: [Error: unexpected end of statement]
[Near : {... ( Item.Name in Delivery_32Items ) ....}]
                                              ^
[Line: 1, Column: 34]
	at org.mvel2.compiler.AbstractParser.nextToken(AbstractParser.java:1286)
	at org.mvel2.MVELInterpretedRuntime.parseAndExecuteInterpreted(MVELInterpretedRuntime.java:92)
	at org.mvel2.MVELInterpretedRuntime.parse(MVELInterpretedRuntime.java:58)
	at org.mvel2.MVEL.eval(MVEL.java:171)
	at org.kie.yard.core.MVELLiteralExpressionInterpreter.fire(MVELLiteralExpressionInterpreter.java:55)
	... 5 more
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 33 out of bounds for length 33
	at org.mvel2.ast.Fold.<init>(Fold.java:73)
	at org.mvel2.compiler.AbstractParser.nextToken(AbstractParser.java:1044)
	... 9 more
```
It's another issue in mvel2. https://github.com/mvel/mvel/commit/043303ce was a valid fix, but it revealed a latent bug in Fold.java.

Need to fix mvel2 and release a new version before drools consumes it.

--- a/build-parent/pom.xml
+++ b/build-parent/pom.xml
@@ -125,7 +125,7 @@
     <version.org.jpmml.model>1.6.4</version.org.jpmml.model> <!-- jpmml-model BSD 3C license - ATTENTION 1.5.1 intentional, because 1.5.1 evaluators works with 1.5.1 -->
     <version.org.junit.jupiter>6.0.3</version.org.junit.jupiter>
     <version.org.junit.platform>6.0.3</version.org.junit.platform> <!-- JUnit 6 uses a unified version across all modules -->
-    <version.org.mvel>2.5.3.Final</version.org.mvel>
+    <version.org.mvel>2.5.4.Final</version.org.mvel>
     <version.org.powermock>2.0.9</version.org.powermock>
     <version.org.slf4j>2.0.17</version.org.slf4j>
     <!-- simple-jndi is a small library that helps us avoid JNDI error messages during testing -->

